pkg/operator/operator.go

```diff
@@ -187,13 +188,13 @@ func (optr *Operator) sync(key string) error {
 	return optr.syncAll(*operatorConfig)
 }
 
-func (optr *Operator) maoConfigFromInstallConfig() (*OperatorConfig, error) {
-	installConfig, err := getInstallConfig(optr.kubeClient)
+func (optr *Operator) maoConfigFromInfrastructure() (*OperatorConfig, error) {
+	infra, err := optr.osClient.ConfigV1().Infrastructures().Get("cluster", metav1.GetOptions{})
```

Comment:
@abhinavdahiya e2e seems to be consistently failing, can't see anything meaningful in the logs. who/when creates the infra object?

Reply:
Managed to debug this by running installer locally from CI payload and running bastion to ssh into masters.
```
E0401 09:48:01.939450       1 operator.go:177] Failed getting operator config: infrastructures.config.openshift.io "cluster" is forbidden: User "system:serviceaccount:openshift-machine-api:default" cannot get resource "infrastructures" in API group "config.openshift.io" at the cluster scope
```
See if it goes green here https://github.com/openshift/machine-api-operator/pull/264/commits/df1d9c076b12eb42d36aa76b7f24a570a7938f19